From b234472acdeb6e41335a4478aec2ce185cad192e Mon Sep 17 00:00:00 2001
From: Wenyou Yang <wenyou.yang@atmel.com>
Date: Mon, 3 Dec 2012 12:56:34 +0800
Subject: [PATCH 158/319] watchdog/at91sam9_wdt: Remove the options:
 WDIOF_SETTIMEOUT and WDIOF_MAGICCLOSE

Since the Watchdog Timer Mode Register can be only written only once,
so struct watchdog_info doesn't support WDIOF_SETTIMEOUT
and WDIOF_MAGICCLOSE options

Signed-off-by: Wenyou Yang <wenyou.yang@atmel.com>
---
 drivers/watchdog/at91sam9_wdt.c |    3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/drivers/watchdog/at91sam9_wdt.c b/drivers/watchdog/at91sam9_wdt.c
index 93ae4e4..7ed3264 100644
--- a/drivers/watchdog/at91sam9_wdt.c
+++ b/drivers/watchdog/at91sam9_wdt.c
@@ -152,8 +152,7 @@ static int at91wdt_enable(struct watchdog_device *wddev, unsigned int timeout)
 
 static const struct watchdog_info at91_wdt_info = {
 	.identity	= DRV_NAME,
-	.options	= WDIOF_SETTIMEOUT | WDIOF_KEEPALIVEPING |
-						WDIOF_MAGICCLOSE,
+	.options	= WDIOF_KEEPALIVEPING,
 };
 
 static int at91wdt_start(struct watchdog_device *wddev)
-- 
1.7.10.4

